src/olympia/lib/cache.py

```diff
@@ -82,53 +80,6 @@ def get(self, delete=False):
         return res
 
 
-class Token(object):
```

Comment:
`Message` above can probably be removed as well, but this can be done as a follow-up (looks like most of this class can go really, `assert_cache_requests` and `CacheStatTracker` are only used in like one test, should be possible to replace with something even simpler)

Reply:
Removed `Message` class in f5122de500.